flowey/flowey_lib_hvlite/src/build_openhcl_igvm_from_recipe.rs

```diff
@@ -80,7 +80,7 @@ pub struct OpenhclIgvmRecipeDetailsLocalOnly {
     pub custom_extra_rootfs: Vec<PathBuf>,
 }
 
-#[derive(Debug, Serialize, Deserialize, Clone)]
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
```

Comment:
once you resolve some other comments, I believe the need for these derives will go away